packages/client/tests/functional/_utils/setupTestSuiteClient.ts

```diff
@@ -119,27 +111,36 @@ export function setupTestSuiteClientDriverAdapter({
   }
 
   if (providerFlavor === ProviderFlavors.JS_PG) {
-    const pool = new pgPool({
+    const { Pool } = require('pg') as typeof import('pg')
```

Comment:
It's this `as typeof import` actually required? (no pun intended)

Reply:
I will try to see if I can put it as a type import instead

Reply:
`require` is not typed for packages, I was surprised